cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -133,6 +134,26 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_index_with_where_using_multiple_range_comparisons() {
```

Comment:
Nice addition: this complements the other PR (as that one didn't have multiple range comparisons). Nothing to do here: just a comment.